core/src/main/java/com/sequenceiq/cloudbreak/service/upgrade/rds/RdsUpgradeService.java

```diff
@@ -96,9 +123,25 @@ private RdsUpgradeV4Response validateAndTrigger(StackView stack, TargetMajorVers
             LOGGER.warn("Stack {} is not available for RDS upgrade", stack.getName());
             throw new BadRequestException(getMessage(CLUSTER_RDS_UPGRADE_NOT_AVAILABLE));
         }
+        validateRequestTime(stack, accountId);
```

Comment:
I would rename it to something that reflects the code is checking for running attached DHs.
Like validateAttachedDatahubsAreNotRunning()